### PR TITLE
feat(LandingPage): added logs to the landing page

### PR DESCRIPTION
**Pull request by RoeiRom**

### CODE REVIEW

File: client/src/components/App/App.tsx [hunk 1]
```diff
@@ -38,29 +38,75 @@ const App: React.FC = (): JSX.Element => {
     const [isUserUpdated, setIsUserUpdated] = useState<boolean>(true);
 
     const setCurrentAndGroupUsers = (userId: string, userName: string, userToken: string) => {
+        logger.info({
+            service: Service.CLIENT,
+            severity: Severity.LOW,
+            workflow: 'Getting group users',
```
Comment (sandymash) [inline client/src/components/App/App.tsx:44]: commented on other PRS, but is there a reason to log so many info logs on client side? 
we already have the same logs on the server, and if the request fails we can still log the error? 🤔 🤷‍♀️

File: client/src/components/App/App.tsx [hunk 2]
```diff
@@ -38,29 +38,75 @@ const App: React.FC = (): JSX.Element => {
     const [isUserUpdated, setIsUserUpdated] = useState<boolean>(true);
 
     const setCurrentAndGroupUsers = (userId: string, userName: string, userToken: string) => {
+        logger.info({
+            service: Service.CLIENT,
```
[the remark this answers is not in the document]
Comment (RoeiRom) [reply client/src/components/App/App.tsx:42]: תכלס זה משהו שבאמת חשבתי עליו רק בדיעבד, האם זה משהו שדחוף לPR הזה או שאפשר לחכות לPR הבא של הלוגים?

File: client/src/components/App/App.tsx (hunk 1 above)
Comment (RoeiRom) [reply client/src/components/App/App.tsx:44]: So we'd have the whole flow, from client to server monitored

File: client/src/components/App/App.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (sandymash) [reply client/src/components/App/App.tsx:42]: ok 🤷‍♀️

File: client/src/components/App/App.tsx (hunk 1 above)
Comment (sandymash) [reply client/src/components/App/App.tsx:44]: I just think its redundant and makes the code and the logs harder to read

Comment (sandymash) [reply client/src/components/App/App.tsx:44]: but if its what was decided then sure

**Review by daniel7764 — DISMISSED** [review]

File: client/src/components/App/App.tsx (hunk 1 above)
Comment (RoeiRom) [reply client/src/components/App/App.tsx:44]: I think this is what is has been decided, but we can bring it up again

**Review by eladyashar — DISMISSED** [review]

**Review by eladyashar — APPROVED** [review]

**Review by MayaSasson — APPROVED** [review]

**Pull request merged**